en/docs/references/test-your-service-via-postman.md

```diff
@@ -0,0 +1,145 @@
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+In this step, you will be creating a collection. The collection runner in Postman runs multiple requests in a specified sequence and logs the results. Each collection represents one request. For more information about collections, see [Postman Documentation - Collection Runner](https://learning.postman.com/docs/running-collections/intro-to-collection-runs/)
+
+!!! tip
+    If you linked an existing workspace to the service and that workspace has existing collections, you can skip this step.
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `invoke-covid-status service`         |
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** panel.
+
+## Step 5: Create a request
+
+In this step, let's create a request that you can send via Postman to invoke your service
+
+To create a request, follow this procedure:
+
+1. In the **Postman** panel, click on the **>** icon of the collection that you created. This opens the **Requests** section.
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `covid-status-request-1`)
+
+4. Select the required HTTP method. You can also add the query parameters, headers, and the request body as required.
+
+    In this example, you can select **GET** as the HTTP method, and add `stats/{country}` as the path.
+    
+    ![Postman request](../assets/img/postman/postman-request.png){.cInlineImage-half}
+    
+5. Click **Save**.
+
+## Step 6: Send a request from your Postman account
+
+To test your service, send a request from your Postman account by following this procedure:
+
+1. Sign in to your Postman account and check your workspaces. The workspace you created via Choreo will be available as shown below.
+
+    ![View workspace in Postman](../assets/img/postman/workspace-in-postman.png){.cInlineImage-half}
+
+2. Click on the workspace. You can view the collection and the request as follows.
+
+    ![Collection and request](../assets/img/postman/collection-and-request.png){.cInlineImage-full}
+    
+3. In the request path displayed, replace `{country}` with `USA` as shown below:
+
+    ![Update request path](../assets/img/postman/update-request-path.png){.cInlineImage-full}
+    
+4. Click **Send**.
+
+    The response is logged as shown below.
+
+    ![Response](../assets/img/postman/response.png){.cInlineImage-full}
```

Comment:
Since this looks like a step by step guide don't we need to have ending sentence saying you have completed xxx.

Reply:
Added a closing sentence.